File: Exemplos M10/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Sat/SatCommand.cs
using Newtonsoft.Json;

namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Sat
{
    abstract class SatCommand : IntentDigitalHubCommand
    {
        //O retorno dos comandos SAT é sempre uma String
        [JsonProperty(PropertyName = "resultado")]
        private readonly string resultado;

        protected SatCommand(string functionName) : base(functionName, IntentDigitalHubModule.SAT) { }

        public string GetResultado()
        {
            return resultado;
        }
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/SAT/sat_command.dart
import 'package:intent_digital_hub/IntentDigitalHubService/intent_digital_hub_command_starter.dart';
import 'package:intent_digital_hub/IntentDigitalHubService/intent_digital_hub_module.dart';

abstract class SatCommand extends IntentDigitalHubCommand {
  SatCommand(String functionName)
      : super(functionName, IntentDigitalHubModule.SAT);
}
